### ISSUE TITLE
Turning hands-free off mutes the mike

### ISSUE THREAD

**Issue by rma-x** [iNPUTmice/Conversations#3706]
#### General information

* **Version:** 2.8.1
* **Device:** Zuk Z1
* **Android Version:**  Cyanogen OS 12.1, based on Android 5.1.1 (stock)
* **Server name:** self hosted
* **Server software:** prosody 0.11.5
* **Installed server modules:** All that Conversations' "Server Info" checks for 
* **Conversations source:** F-Droid

#### Steps to reproduce

1. Make an audio call (incoming or outgoing doesn't matter)
2. Disable hands-free speaking by tapping on the speaker icon until it is stroked-trough.

#### Expected result

Speaker volume goes down, so that I can talk to the other party by holding the phone to my ear.

#### Actual result

Speaker volume does go down and I can still hear the other party, but the other party doesn't hear me anymore as if my microphone was muted although the microphone icon still shows it in unmuted state. Going back to hands-free mode also unmutes my microphone again.

Debug output can be provided on rquest.

**Comment by iNPUTmice** [comment]
Conversations usually starts in Earpiece mode. So the steps to reproduce seem to be missing the initial step of going into hands-free mode in the first place and the important information if the mic works then.

**Comment by rma-x** [comment]
Whenever Conversations is in earpiece mode (also initially) the mic is muted and whenever it is in hands-free mode the mic works.

**Comment by Roxxor91** [comment]
>     * **Android Version:**  Cyanogen OS 12.1, based on Android 5.1.1 (stock)
I am really sorry for the offtopic, I would have written a PM if it was possible on Github (it's not, right?)
Update your Android Version! This is ancient! Your device is exploitable by many methods. Many of the fixed vulnerabilities were at the level of eg. remote code execution by simply opening/downloading .png files.
Your device is well supported by the community: https://forum.xda-developers.com/zuk-z1/orig-development

Again, sorry for offtopic.

**Comment by rma-x** [comment]
Yeah, switching this to LineageOS has been on my todo list ever since they added support for it, but I never took the time to actually do it, including reinstallation and reconfiguration of everything...
But thanks for kicking me in the ..., I will reconsider it now.
